validate.go

```diff
@@ -20,9 +20,13 @@ func (validator *Validator) Validate(subject interface{}) (bool, ViolationList,
 	return validator.ValidateWithContext(context.Background(), subject)
 }
 
-func (validator *Validator) ValidateWithContext(ctx context.Context, subject interface{}) (bool, ViolationList, error) {
-	var violations ViolationList
-	var err error
+func (validator *Validator) ValidateWithContext(ctx context.Context, subject interface{}) (isValid bool, violations ViolationList, err error) {
+
+	defer func() {
+		if r := recover(); r != nil {
```

Comment:
I am just wondering if we should also set `isValid` to false in case of a panic. I thin that would be more consistent as we also return false with the non nil error.

Reply:
It wouldn't be necessary since the default value of isValid is false.